src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/ConfigurationLiveTests.cs

```diff
@@ -412,10 +412,9 @@ public async Task Revisions()
                 await service.SetAsync(testSettingUpdate);
 
                 // Test
-                var filter = new BatchRequestOptions();
-                filter.Key = setting.Key;
-                filter.Revision = DateTimeOffset.MaxValue;
-                SettingBatch batch = await service.GetRevisionsAsync(filter, CancellationToken.None);
+                var selector = new ConfigurationSelector(setting.Key);
+                selector.AcceptDateTime = DateTimeOffset.MaxValue;
```

Comment:
AcceptDateTime does not tell me whether it is exact time, any before, or any after. Would it be possible to come up with a name that makes it clear? 

cc: @johanste

Reply:
The exact header is Accept-DateTime and the semantics are described [here](https://tools.ietf.org/html/rfc7089#section-2.1.1). We can choose to pick a standard parameter name (or method name) to describe the semantic across all of our libraries (maybe `AsOf`?), or we can stick with something that is closer to the HTTP nomenclature with the assumption that customers can map between the two.

Reply:
Yeah, "AsOf" would make it clearer, but it does depart from HTTP terminology. Maybe we should have a quick discussion on this at one of the arch meetings?

Reply:
I can change it to `AsOf` fro now and keep pushing for a conclusion until we like a name

Reply:
FYI @conniey @xiangyan99  @YijunXieMS